lib/beebee/storage/redis.ex

```diff
@@ -49,6 +49,29 @@ defmodule BeeBee.Storage.Redis do
     end
   end
 
+  @impl true
+  def update(url, short_tag) do
+    with {:ok, 1} <- Redix.command(@process_name, ["EXISTS", url_key(short_tag)]),
+         {:ok, ["OK"]} <-
+           Redix.pipeline(@process_name, [
```

Comment:
You can probably just use `Redix.command` here instead of `pipeline` as there's only one command to execute.